docs/index.md

```diff
@@ -1307,6 +1311,9 @@ The `Application` object is instantiated with the following settings:
 - `types`: A list of all resource types declared and handled by this app.
 - `processors`: If you define custom processors, they have to be registered here as instances.
 - `defaultProcessor`: All non-bound-to-processor resources will be handled by this processor.
+- `transportLayerOptions`: Allows to customize some aspects of the transport layer used by the application.
+  - `httpBodyPayload`: Describes how big the request's `body` can be, expressed in a string i.e. `10mb`. Defaults to `1mb`.
+  - `httpStrictMode`: If enabled, requires all HTTP incoming requests to use `Content-Type: application/vnd.api+json`. Defaults to `false`.
```

Comment:
I had an idea of global `strictMode: true` that could be used for multiple things, not just for content-type.
For example, profiles should also be included in content-type 
`Content-Type: application/vnd.api+json; profile="https://example.com/resource-timestamps"`
https://jsonapi.org/format/1.1/#profile-rules

Extensions also:
`Content-Type: application/vnd.api+json; ext="https://jsonapi.org/ext/version"`
https://jsonapi.org/format/1.1/#extension-rules

We might find other loose rules in the future and throw an error in strict mode if they are not followed.

Reply:
For now, the only thing we check in `strictMode` is the Content-Type. Eventually we can keep adding more features to it. We do not support profiles or extensions at present, so we can make the necessary adjustments in that moment, I think.